src/lib/homestore_backend/tests/test_shard_manager.cpp

```diff
@@ -104,26 +109,7 @@ TEST_F(ShardManagerTesting, ListShardsOnEmptyPg) {
     e.then([this](auto const& info_list) { ASSERT_EQ(info_list.size(), 0); });
 }
 
-class ShardManagerWithShardsTesting : public ShardManagerTesting {
-    // std::shared_ptr< home_replication::ReplicaSetListener > _listener;
-
-public:
-    void SetUp() override {
-        ShardManagerTesting::SetUp();
-#if 0
-        homeobject::HSHomeObject* ho = dynamic_cast< homeobject::HSHomeObject* >(_home_object.get());
-        EXPECT_TRUE(ho != nullptr);
-        auto rs = ho->get_repl_svc()->get_replica_set(fmt::format("{}", _pg_id));
-        EXPECT_TRUE(std::holds_alternative< home_replication::rs_ptr_t >(rs));
-        auto replica_set =
-            dynamic_cast< home_replication::MockReplicaSet* >(std::get< home_replication::rs_ptr_t >(rs).get());
-        _listener = std::make_shared< homeobject::ReplicationStateMachine >(ho);
-        replica_set->set_listener(_listener);
-#endif
-    }
-};
-
-TEST_F(ShardManagerWithShardsTesting, CreateShardSuccess) {
+TEST_F(ShardManagerTesting, CreateShardSuccess) {
```

Comment:
We need a test case with one PG, and create more than one Shard inside this PG. The 2nd and later shards created in this PG should arrive on chunk which belongs to same pdev_id. 

Also can we create 2 PGs and each PG with more than 1 shard created?

Reply:
such test cases are added in the latest pr.